index.html

```diff
@@ -565,6 +565,7 @@
 The time immediately after the user agent receives the last byte of the response or immediately before the transport connection is closed, whichever comes first. The resource here can be received either from <a

```

Comment:
Not currently. And synthetic responses / content codings make this header unreliable.

If user agents were to implement such logic we might add it, but it seems that for something that streams for a long time, it would be a rather disastrous user experience.